public/profile.html

```
@@ -0,0 +1,601 @@
+    </div>
+  </div>
+  <div style="padding:7px;">
+    <a href="/profile.html" style="display:flex;align-items:center;gap:10px;padding:9px 11px;border-radius:9px;color:var(--teal);font-size:.83rem;font-weight:600;background:var(--teal-dim);">
+      <svg width="15" height="15" viewBox="0 0 16 16" fill="none"><circle cx="8" cy="5" r="3" stroke="currentColor" stroke-width="1.4"/><path d="M2 14c0-3.314 2.686-5 6-5s6 1.686 6 5" stroke="currentColor" stroke-width="1.4" stroke-linecap="round"/></svg> View Profile
+    </a>
+    <a href="/dashboard.html" style="display:flex;align-items:center;gap:10px;padding:9px 11px;border-radius:9px;color:var(--ink);font-size:.83rem;font-weight:500;transition:background .15s;margin-top:2px;" onmouseover="this.style.background='#F0FDFA'" onmouseout="this.style.background='transparent'">
+      <svg width="15" height="15" viewBox="0 0 16 16" fill="none"><rect x="2" y="2" width="5" height="5" rx="1.5" stroke="#0D0D14" stroke-width="1.4"/><rect x="9" y="2" width="5" height="5" rx="1.5" stroke="#0D0D14" stroke-width="1.4"/><rect x="2" y="9" width="5" height="5" rx="1.5" stroke="#0D0D14" stroke-width="1.4"/><rect x="9" y="9" width="5" height="5" rx="1.5" stroke="#0D0D14" stroke-width="1.4"/></svg> Dashboard
+    </a>
+  </div>
+  <div style="padding:7px;border-top:1px solid rgba(13,13,20,.07);">
+    <button onclick="logout()" style="display:flex;align-items:center;gap:10px;padding:9px 11px;border-radius:9px;background:transparent;border:none;cursor:pointer;color:#e53e3e;font-size:.83rem;font-weight:500;width:100%;text-align:left;transition:background .15s;" onmouseover="this.style.background='#fff5f5'" onmouseout="this.style.background='transparent'">
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)